.common-ci.yml

```diff
@@ -172,6 +172,14 @@ trigger-pipeline:
     - if [[ "${PRECOMPILED}" == "true" ]]; then ./scripts/precompiled.sh push; else make push-${DIST}-${DRIVER_VERSION}; fi
 
 .release-ubuntu22.04:
+  # Perform for each DRIVER_VERSION
+  extends:
+    - .release-generic
+    - .driver-versions-ubuntu24.04
```

Comment:
This block is under the `.release-ubuntu22.04` by mistake I believe.

Reply:
d'oh. Thanks

Reply:
Fixed